src/main/resources/application.yml

```diff
@@ -0,0 +1,3 @@
+spring:
+  profiles:
+    active: development
```

Comment:
You already have application-development.yml properties for the development profile, pls delete it